smoke/src/main/java/com/adobe/cq/cloud/testing/it/smoke/GetTogglesIT.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2019 Adobe Systems Incorporated
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.adobe.cq.cloud.testing.it.smoke;
+
+import com.adobe.cq.testing.client.CQClient;
+import com.adobe.cq.testing.junit.rules.CQAuthorPublishClassRule;
+import com.adobe.cq.testing.junit.rules.CQRule;
+import com.adobe.cq.testing.junit.rules.Page;
+import org.apache.sling.testing.clients.ClientException;
+import org.junit.*;
+
+public class GetTogglesIT {
+
+    @ClassRule
+    public static CQAuthorPublishClassRule cqBaseClassRule = new CQAuthorPublishClassRule();
+
+    @Rule
+    public CQRule cqBaseRule = new CQRule(cqBaseClassRule.authorRule, cqBaseClassRule.publishRule);
+
+    @Rule
+    public Page root = new Page(cqBaseClassRule.authorRule);
+
+    static CQClient adminAuthor;
+    static CQClient adminPublish;
+
+    @BeforeClass
+    public static void beforeClass() throws ClientException {
+        adminAuthor = cqBaseClassRule.authorRule.getAdminClient(CQClient.class);
+
+        adminPublish = cqBaseClassRule.publishRule.getAdminClient(CQClient.class);
+    }
+
+    /**
+     * Verifies that the homepage exists on author
+     */
+    @Test @Ignore
```

Comment:
Why ignore?

Reply:
We're ignoring these tests because the published Toggle Router still uses the token that isn't easy to extract (could we use some helper from the `com.adobe.cq.testing` library to get it?) and will fail if run.